includes/class-wc-stripe-helper.php

```
@@ -1460,6 +1460,64 @@ public static function add_payment_method_to_request_array( string $payment_meth
+				sprintf(
+					'Removed %1$s (value: %2$s) from an intent request. It was added by third-party code hooking wc_stripe_generate_create_intent_request. This store is connected via Stripe App OAuth, so the fee would have been collected by the plugin platform account rather than by this store.',
+					$field,
+					wp_json_encode( $request[ $field ] )
+				)
+			);
+
+			unset( $request[ $field ] );
+		}
+
+		return $request;
+	}
```

PHPMD (2.15.0)

[error] The method strip_platform_fee_fields is not named in camelCase. (undefined)

(CamelCaseMethodName)